#
#

float32 steering_wheel_torque   # desired torque on steering wheel (in percentages [-100:100])

#
#

float32 throttle                # desired vehicle acceleration (in percentages [-1000:1000])
